contributions/contribution.yaml

```
@@ -483,6 +483,30 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /contributions/version:
+    get:
+      summary: Get version of logging building block
+      tags:
+        - Contributions
+      responses:
+        200:
+          description: The version of logging building block.
+        400:
+          description: Bad request
+        500:
+          description: Internal server error
+  /contributions/buildingblocks:
```

```suggestion
  /contributions/building-blocks:
```
Let's use a hyphen to split the words like /profiles/device-data. Please make necessary changes in Python code too. Thanks.

---

changed